library/Fission/CLI/IPFS/Connect.hs

```diff
@@ -7,7 +7,7 @@ module Fission.CLI.IPFS.Connect
 import           Fission.Prelude
 
 import qualified System.Console.ANSI as ANSI
-import           Data.List.NonEmpty as NonEmpty hiding ((<|))
+import           RIO.NonEmpty as NonEmpty hiding ((<|))
```

Comment:
If you import it qualified (which is how you’re using it AFAICT), you can sidestep conflicting imports

```haskell
import qualified RIO.NonEmpty as NonEmpty
```